tests/test_skills/test_apy_estimation_abci/test_apis.py

```diff
@@ -97,7 +102,44 @@ def test_eth_price(spooky_specs: SpecsType, eth_price_usd_q: str) -> None:
         res = make_request(api.get_spec(), eth_price_usd_q)
         eth_price = ast.literal_eval(api.process_response(DummyMessage(res.content))[0]["ethPrice"])  # type: ignore
 
-        assert isinstance(eth_price, float)
+        assert eth_price == 0.4183383786296383
```

Comment:
We should have that price defined in a constant.

Reply:
Done in 65763b23c6c6c3d5c2f4ab662999c8d303602b72.